src/components/pager/pager.styles.js

```diff
@@ -0,0 +1,119 @@
+import styled, { css } from 'styled-components';
+import { THEMES } from '../../style/themes';
+
+const PagerContainerStyles = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding: 9px 24px;
+  align-items: center;
+  border: 1px solid #d8dfe3;
+  border-top-width: 0;
+  font-size: 13px;
+  background-color: '#FAFBFB';
```

Comment:
please use theme variables instead of hardcoded colors values

Reply:
There's a bit of cross-over with the `Table` component, leaving hard-coded colours for the moment, likely to take the lead from `Table`.